### PR TITLE
Allow build with qt6

### PR DESCRIPTION
**Pull request by neochapay**

### CODE REVIEW

File: src/seasidevcardmodel.cpp [hunk 1]
```diff
@@ -257,12 +257,13 @@ void SeasideVCardModel::readContacts()
     if (vcf.open(QIODevice::ReadOnly)) {
         // TODO: thread
         QVersitReader reader(&vcf);
-
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
         if (!mDefaultCodec.isEmpty()) {
             QTextCodec *codec = QTextCodec::codecForName(mDefaultCodec.toLatin1());
```
Comment (rainemak) [inline src/seasidevcardmodel.cpp:262]:
```suggestion
            QByteArray readerCodec = mDefaultCodec.toLatin1();
            QTextCodec *codec = QTextCodec::codecForName(readerCodec.constData());
```

This way this surrounding QT_VERSION check could be removed.

Qt5.6: https://doc.qt.io/archives/qt-5.6/qtextcodec.html#codecForName-1
Qt6: https://doc.qt.io/qt-6/qtextcodec.html#codecForName-1

**Review by rainemak — COMMENTED** [review]
Minor remark added. This way one version check could be avoided and both Qt5.6 and Qt6 would work exactly same way.

File: src/seasidevcardmodel.cpp (hunk 1 above)
Comment (neochapay) [reply src/seasidevcardmodel.cpp:262]: QTextCodec in core5compat and not good to use it

**Review by rainemak — APPROVED** [review]
LGTM

**Pull request merged**